### ISSUE TITLE
Importing Xmind 2021 files do not work

### ISSUE THREAD

**Issue by mbledkowski** [phase1geo/Minder#403]
When I try to open xmind file created in Xmind 2021 this is what shows up.
![Screenshot from 2021-08-21 03-10-00](https://user-images.githubusercontent.com/23295125/130305857-93e966d8-4eb6-4bf3-9c79-50226cbe25c8.png)

**Comment by phase1geo** [comment]
I believe XMind 8 is supported. I haven’t done any work with XMind 2021. I can take a look, but the format differences might take more than a quick fix.

**Comment by phase1geo** [comment]
Looks like the newer version of XMind reproduces JSON formatted data files instead of XML. I’m working on support for this newer version in addition to still supporting XMind 8 output.

**Comment by phase1geo** [comment]
In the master branch, I have committed initial support for importing the newer version of XMind.  I have also attempted to add export support; however, XMind 2020 is not able to open the generated file, so I have disabled export support for the time being.  I'll tinker around with the file format work.  Not sure if I can properly solve this or not since I don't see any documentation for the newer file format, but I'll keep the action open for now.

Let me know if you see any problems with the import functionality.
